### PR TITLE
Update readme and CODEOWNERS

### PR DESCRIPTION
**Pull request by fdegir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ The process SIG Interoperability follows can be seen from [here](https://github.
 Chairs and the TOC Sponsor of the SIG are
 
 * Melissa McKay ([@mjmckay](https://github.com/mjmckay)), JFrog - Chair
-* Kara de la Marck ([@MarckK](https://github.com/MarckK)), Continuous Delivery Foundation - Chair
+* Dadisi Sanyika([@dsanyika](https://github.com/dsanyika)), Apple - Chair
```
Comment (afrittoli) [inline README.md:122]: Shall we start an emeritus list?

File: README.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ The process SIG Interoperability follows can be seen from [here](https://github.
...
 * Dan Lorenc ([@dlorenc](https://github.com/dlorenc)), Google - TOC Sponsor
```
Comment (afrittoli) [inline README.md:123]: @dlorenc are you still involved in this group?

**Review by oleg-nenashev — APPROVED** [review]
+1 for Emeritust

File: README.md (hunk 1 above)
Comment (fdegir) [reply README.md:122]: That's a good idea. Let me send a new commit and add a section similar to the one in TOC readme.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # These owners will be the default owners for everything in
 # the repo. People listed below will be requested for review
 # when someone opens a pull request.
-* @mjmckay @amfred @e-backmark-ericsson @olivernocon @afrittoli @thedevelopnik @rakhbari @bobcatfish @fdegir @MarckK @zxiiro @tracymiranda @agrimmer @justinabrahms
+* @mjmckay @amfred @e-backmark-ericsson @olivernocon @afrittoli @thedevelopnik @rakhbari @bobcatfish @fdegir @MarckK @zxiiro @tracymiranda @agrimmer @justinabrahms @dsanyika
```
Comment (justinabrahms) [inline .github/CODEOWNERS:4]:
```suggestion
* @mjmckay @amfred @e-backmark-ericsson @olivernocon @afrittoli @thedevelopnik @rakhbari @bobcatfish @fdegir @MarckK @zxiiro @tracymiranda @agrimmer @dsanyika
```

**Review by justinabrahms — APPROVED** [review]

**Pull request merged**